addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -1861,6 +1862,32 @@ QUnit.module("Fields", (hooks) => {
             await closePopover(target);
             assert.strictEqual(property.querySelector(".o_property_field_value input").value, "");
         }
+
+        // add a new date property
+        await click(target, ".o_field_property_add button");
```

Comment:
I would do that in another test. We try to have more, but smaller tests which are easier to understand/maintain